packages/react/src/Switch/Switch.tsx

```diff
@@ -0,0 +1,31 @@
+import { ForwardedRef, forwardRef, InputHTMLAttributes, PropsWithChildren, useId } from 'react'
+
+export const Switch = forwardRef(
+  (
+    { id, ...restProps }: PropsWithChildren<InputHTMLAttributes<HTMLInputElement>>,
+    ref: ForwardedRef<HTMLInputElement>,
+  ) => {
+    const inputId = id || useId()
+
+    return (
+      <div className="amsterdam-switch">
+        <input
+          {...restProps}
+          className="amsterdam-switch__input"
+          id={inputId}
+          ref={ref}
+          type="checkbox"
+          role="switch"
+        />
+
+        <label className="amsterdam-switch__label" htmlFor={inputId} aria-hidden="true">
```

Comment:
Is `aria-hidden` necessary here? My screen reader doesn't announce anything for empty labels.